includes/jobqueue/JobQueueGroup.php

```diff
@@ -237,6 +237,11 @@ public function lazyPush( $jobs ) {
 		// Throw errors now instead of on push(), when other jobs may be buffered
 		$this->assertValidJobs( $jobs );
 
+		/** Fandom change - start (@author wbodzek) - hook for logging */
+		MediaWikiServices::getInstance()->getHookContainer()
```

Comment:
Just wanted to confirm that we've checked that there's no way of implementing changes in UCP only? since we are patching core only if nothing else can be done

Reply:
No other hooks around. Moreover it might be removed later when we gather more insight.